### DISCUSSION TITLE
How to use Get-UiPathAuthToken when using an application id and application secret

### DISCUSSION THREAD

**Discussion by taimoor-hotiana-bzy** [UiPath/orchestrator-powershell#175] [Q&A]
Hello, I'm struggling with the syntax to use Get-UiPathAuthToken when using a registered external app (using application id and application secret) and using Azure Authentication on Cloud Orchestrator.

I can see there is an option to use -CloudDeployment parameter but no examples. Can someone who has done it please give an example powershell?

**Comment by mSchirru** [comment]
@taimoor-hotiana-bzy did you manage to figure out something?
